packages/clients/dish/src/types.ts

```diff
@@ -124,3 +126,12 @@ export interface DishMapConfig
     urlParams: DishUrlParams
   }
 }
+
+export interface SelectionObjectState {
+  objectId: number
```

Comment:
```suggestion
  objectId: string
```
While a string containing a number, this still seems to always be a string – the `ObjektID` is taken from the parameters, but never parsed to become an actual number. It still works, but may confuse someone in future iterations.